docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -237,14 +276,86 @@ class DebugPage extends React.PureComponent<Props, State> {
 
   private onCopy = () => {
     const response = this.state.debugResponse;
-    const textArea = document.createElement('textarea');
-    textArea.style.opacity = '0.0';
-    textArea.value = response;
-    document.body.appendChild(textArea);
-    textArea.focus();
-    textArea.select();
-    document.execCommand('copy');
-    document.body.removeChild(textArea);
+    if (response.length > 0) {
+      DebugPage.copyTextToClipboard(response);
+      this.showSnackbar('The response has been copied to the clipboard.');
+    }
+  };
+
+  private onExport = () => {
+    const escapeSingleQuote = (text: string) => text.replace(/'/g, `'\\''`);
+
+    const endpointPath = this.state.endpointPath; // annotated service only
```

Comment:
This line can be removed and the `endpointPath` can be declared where it needs (line 330).

Reply:
Thanks :)